Examples/UIExplorer/TouchableExample.js

```diff
@@ -111,6 +111,17 @@ exports.examples = [
    },
  }];
 
+if(Platform.OS === 'android') {
```

Comment:
You can add a `platform` prop to the example object instead of checking the platform manually. See the '3D Touch' example above that is ios only.